kuma/authkeys/migrations/0003_remove_authkeys.py

```diff
@@ -0,0 +1,76 @@
+# -*- coding: utf-8 -*-
+import datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        "Write your forwards methods here."
+        orm.Key.objects.all().delete()
```

Comment:
Good point. When I ran it locally it deleted them automatically via the default cascade behavior - is that good enough?

But this reminds me that there is a `Key` with existing `KeyAction` records. Should I change this migration to only delete Keys that have not performed any actions? That will preserve the existing API action log.